src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [filterName, setFilterName] = useState('');
+
+  const visibleMovies = moviesFromServer
+    .filter(movie => movie.title.toLowerCase()
+      .includes(filterName.toLowerCase())
+    || movie.description.toLowerCase()
+      .includes(filterName.toLowerCase()));
```

Comment:
decompose this logic

Reply:
Create another variables? Don't understand what exactly I need to decompose

Reply:
creating a var is one of the possible ways to make it more readable and maintainable
why not to create some function-helper? as an example
```
const containsString = (...) => ...

const lowercaseQuery = ...

const visibleMovies = movies.fromServer.filter(({ ... }) => {
  const hasTitle = ...
  const hasDescription = ...

  return hasTitle || hasDescription;
});
```

Reply:
is it good?